packages/protocol/foundry.toml

```diff
@@ -34,4 +35,8 @@ fs_permissions = [
     { access = "read", path = "./artifacts/"}
     ]
 
+[profile.devchain] # Special profile for the tests that require an anvil devchain
```

Comment:
I think these lines (hidden in the PR), can probably moved to the devchain toml.

```
   { access = "read", path = "./migrations_sol/migrationsConfig.json"},
    { access = "read", path = "./governanceConstitution.json"},
```

Reply:
They can, but I think it's better to keep the file access info in the default profile. Otherwise, the `FOUNDRY_PROFILE=devchain` has to be passed everytime a migration script is run (e.g. `FOUNDRY_PROFILE=devchain forge script ...`). I don't think that's necessary.

https://github.com/celo-org/celo-monorepo/blob/125391cd0a5b1fcedd79cef89439fde582074053/packages/protocol/foundry.toml#L31-L37